mods/d2k/rules/infantry.yaml

```diff
@@ -253,7 +253,7 @@ saboteur:
 		Queue: Infantry
 		BuildPaletteOrder: 100
 		Prerequisites: ~disabled
-		Description: Sneaky infantry, armed with explosives\n  Strong vs Buildings\n  Weak vs Everything\n  Special Ability: destroy buildings
+		Description: Sneaky infantry, armed with explosives.\nCan be deployed to become invisible for a while.\n  Strong vs Buildings\n  Weak vs Everything\n  Special Ability: destroy buildings
```

Comment:
Please change this to something like "Can be deployed to become invisible for a limited time."